content/quickstart/Node.js/metrics.md

```diff
@@ -0,0 +1,571 @@
+---
+title: "Metrics"
+date: 2018-08-20T07:12:03-13:00
+draft: false
+class: "shadowed-image lightbox"
+---
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Brief Overview](#brief-overview)
+- [Getting started](#getting-started)
+- [Enable Metrics](#enable-metrics)
+    - [Import Packages](#import-packages)
+    - [Create Metrics](#create-metrics)
+- [Record and Aggregate Data](#record-and-aggregate-data)
+    - [Create Views and Tags](#create-views-and-tags)
+    - [Recording Metrics](#recording-metrics)
+- [Exporting to Stackdriver](#exporting-to-stackdriver)
+
+In this quickstart, we’ll gleam insights from code segments and learn how to:
+
+1. Collect metrics using [OpenCensus Metrics](/core-concepts/metrics) and [Tags](/core-concepts/tags)
+2. Register and enable an exporter for a [backend](/core-concepts/exporters/#supported-backends) of our choice
+3. View the metrics on the backend of our choice
+
+## Requirements
+- [Node.js](https://nodejs.org/) 6 or above and `npm` (already comes with Node.js)
+- Google Cloud Platform account and project
+- Google Stackdriver Monitoring enabled on your project
+
+{{% notice tip %}}
+For assistance setting up Stackdriver, [Click here](/codelabs/stackdriver) for a guided codelab.
+{{% /notice %}}
+
+## Installation
```

Comment:
It would be nice to handhold the user throughout getting set up, like so:

<pre>
## Installation

We will first create our project directory and files, retrieve dependencies, and bootstrap our entry file.

```bash
mkdir repl-app
cd repl-app

npm install @opencensus/core
npm install @opencensus/exporter-stackdriver

touch repl.js
touch test.txt
```

Put the following code inside of `repl.js`:
[...]
</pre>

Reply:
Nice, thanks @hvent90!

Reply:
Nice suggestion! Thanks, updated. If you can, please take a look and tell me if there is anything more I can improve.